Sources/Rendering/OpenGL/Texture/index.js

```diff
@@ -288,17 +288,19 @@ function vtkOpenGLTexture(publicAPI, model) {
       publicAPI.getOpenGLFilterMode(model.magnificationFilter)
     );
 
-    model.context.texParameteri(
-      model.target,
-      model.context.TEXTURE_BASE_LEVEL,
-      model.baseLevel
-    );
+    if (model.openGLRenderWindow.getWebgl2()) {
```

Comment:
Can we do something here to set base and max level if we are in a WebGL1 context? Without proper levels, LOD interpolation will fail.

Reply:
In webgl1, you can't set base and max level. There are automatically computed by using input textures. All the missing textures are set to null. Then, it's inside your shader where you have to manage the level that will be use. Hope it's clear enough.